aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -124,7 +125,6 @@ void rsqrt_kernel_cuda(TensorIteratorBase& iter) {
       iter.common_dtype(), "rsqrt_cuda",
       [&]() {
         gpu_kernel(iter, []GPU_LAMBDA(scalar_t a) -> scalar_t {
-          // In CUDA, ::rsqrt is overloaded for float and at::Half here is implicitly cast to float.
           return rsqrt_wrapper(a);
```

Comment:
@ngimel I think it would be better to explicitly cast it to float with `opmath` than relying on implicit casting behavior. wdyt?

Reply:
Hi, @ngimel! Gentle ping :)